app/code/Magento/InventoryApi/Api/Data/StockItemManagerInterface.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryApi\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Interface StockItemManagerInterface
+ * @api
+ */
+interface StockItemManagerInterface extends ExtensibleDataInterface
```

Comment:
why do we need this DataInterface? 
it duplicated StockItemInterface, but called manager and never used

looks like a mistake